LiteFarmOrg/LiteFarm
# [LF-2836] [LF-2826] download a document from the documents home and read only page

To test: 

[LF-2826]
1. Go to the documents listing page.  
2. add a document 
3. check on the screen if you can find the download option.
4. click on the download button to see the uploaded file.

[LF-2836]
1. After adding the document. click on the document tile to go to the document details page.
2. On the preview you can find a download button, try clicking on it and check if the document is downloading.

For more details:
https://lite-farm.atlassian.net/browse/LF-2826
https://lite-farm.atlassian.net/browse/LF-2836


[LF-2826]: https://lite-farm.atlassian.net/browse/LF-2826?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[LF-2836]: https://lite-farm.atlassian.net/browse/LF-2836?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
